content/en/apps/concepts/interoperability.md

```diff
@@ -9,8 +9,8 @@ relatedContent: >
   apps/guides/integrations
 ---
 
-*Interoperability* and *Integration* are often used interchangeably but mean different things. [This](https://www.linkedin.com/pulse/lets-talk-interoperability-vs-integration-mike-fleck) article offers a useful delineation between the two.
+*Interoperability* and *Integration* are often used interchangeably but mean different things. [This article](https://www.linkedin.com/pulse/lets-talk-interoperability-vs-integration-mike-fleck) offers a useful delineation between the two.
 
 With **interoperability**, systems work together, even if they weren’t designed specifically to work together, because standard communication protocol techniques are applied. Two (or more) systems that weren't designed to be interoperable require **integration**.
 
-[OpenHIE](https://ohie.org/), [HL7 FHIR](https://www.hl7.org/fhir/index.html), [ICD-10](https://www.who.int/classifications/icd/factsheet/en/), [LOINC](https://loinc.org/), etc... are examples of healthcare related protocols and standards used to achieve interoperability. CHT Core can be configured to support many of these standards and also exchange data with other systems through *integrations*.
+[OpenHIE](https://ohie.org/), [HL7 FHIR](https://www.hl7.org/fhir/index.html), [ICD-10](https://www.who.int/classifications/icd/factsheet/en/), [LOINC](https://loinc.org/), etc... are examples of healthcare related protocols and standards used to achieve interoperability. CHT Core can be configured to support many of these standards and also exchange data with other systems through [*integrations*]({{< ref "apps/guides/integrations" >}}).
```

Comment:
I probably would have linked to "apps/features/integrations" instead since it's the next level of detail in the concepts/features/examples/tutorials/quick guides/reference hierarchy, but am OK if you want to link directly to guides.

Reply:
Ya, that makes sense!

```suggestion
[OpenHIE](https://ohie.org/), [HL7 FHIR](https://www.hl7.org/fhir/index.html), [ICD-10](https://www.who.int/classifications/icd/factsheet/en/), [LOINC](https://loinc.org/), etc... are examples of healthcare related protocols and standards used to achieve interoperability. CHT Core can be configured to support many of these standards and also exchange data with other systems through [*integrations*]({{< ref "apps/features/integrations" >}}).
```